### PR TITLE
homebank: 5.5.6 -> 5.6.3

### PR DESCRIPTION
**Pull request by frlan**
###### Description of changes

Update 5.5.6 -> 5.6.3

This MR updates also the URL where to download the source package.  There were many versions in between, but the ChangeLog can be found at http://homebank.free.fr/ChangeLog 

This MR will also make #219722 obsolete. 

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
-[x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by drupol — CHANGES_REQUESTED** [review]

File: pkgs/applications/office/homebank/default.nix [hunk 1]
```diff
@@ -3,12 +3,11 @@
...
   pname = "homebank";
-  version = "5.5.6";
+  version = "5.6.3";
   src = fetchurl {
-    url = "http://homebank.free.fr/public/homebank-${version}.tar.gz";
-    sha256 = "sha256-Rg6OjHLkwVIDnXqzqPXA8DxqSdrh2T6V/gLBND8vx9o=";
+    url = "http://homebank.free.fr/public/sources/homebank-${version}.tar.gz";
+    sha256 = "419475f564bbd9be7f4101b1197ce53ea21e8374bcf0505391406317ed823828";
   };
-
```
[the remark this answers is not in the document]
Comment (frlan) [reply pkgs/applications/office/homebank/default.nix:11]: Any link to docu how to generated it? Wasn't able to find it

[the remark this answers is not in the document]
Comment (drupol) [reply pkgs/applications/office/homebank/default.nix:11]: I'm just saying that removing a line here is not required and you should not do it.

**Comment by drupol** [comment]
I'll set this PR in `draft` until things are solved.

**Comment by frlan** [comment]
@drupol  Thanks for the input -- much appreciated!

I'm closing this one and will update with some for .5 minor release

**Comment by drupol** [comment]
Cool, ping me when it's done, I'll review it.

**Pull request closed without merging**